CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 # Change Log
 
+## [0.17.0] 11 June 2024
+
+### Added
+- Added radar sensor simulation
+  - Added API call to processes point cloud to create radar-like output
+    - `rgl_node_points_radar_postprocess`
+  - Added API call to remove ground using RANSAC method to fit the plane model to the point cloud
+    - `rgl_node_points_remove_ground`
+  - Added API call to filter ground based on the incident angle of the ray hitting the mesh triangle
+    - `rgl_node_points_filter_ground`
+  - Added API call to compact point cloud by given field (RGL_FIELD_IS_HIT_I32 or RGL_FIELD_IS_GROUND_I32)
+    - `rgl_node_points_compact_by_field`
+    - At the same time, `rgl_node_points_compact` became deprecated
+  - Added API call to publish [RadarScan](http://docs.ros.org/en/noetic/api/radar_msgs/html/msg/RadarScan.html) message into ROS2 topic
+    - `rgl_node_publish_ros2_radarscan`
+  - Added new fields (point attributes) calculation:
+    - Velocity of the hit point on the entity that depends on entity's linear and angular velocity, and mesh deformations (inferred from calls `rgl_entity_set_pose`, `rgl_scene_set_time` and `rgl_mesh_update_vertices`).
+      - `RGL_FIELD_ABSOLUTE_VELOCITY_VEC3_F32`
+      - `RGL_FIELD_RELATIVE_VELOCITY_VEC3_F32`
+      - `RGL_FIELD_RADIAL_SPEED_F32`
+    - Normal vector of the mesh triangle where the hit-point is located
+      - `RGL_FIELD_NORMAL_VEC3_F32`
+    - Incident angle of the ray hitting the mesh triangle
+      - `RGL_FIELD_INCIDENT_ANGLE_F32`
+    - Elevation angle of the hit point
+      - `RGL_FIELD_ELEVATION_F32`
```

Comment:
Just to be sure, azimuth was already there?

Reply:
It was, but wasn't calculated. I will add them.